persistence/src/main/java/edu/softserve/zoo/persistence/repository/impl/AnimalRepositoryImpl.java

```diff
@@ -0,0 +1,14 @@
+package edu.softserve.zoo.persistence.repository.impl;
+
+import edu.softserve.zoo.model.Animal;
+import edu.softserve.zoo.persistence.repository.AnimalRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * <p>Realisations of all {@link AnimalRepository} specific addons for {@link Animal} entity</p>
```

Comment:
Interesting thing to note: check the difference between realization and implementation in translator